### PR TITLE
added kurtosis function and tests for it

### LINKED ISSUE #16360: Add kurtosis function in stats
Kurtosis is an essential measure in Statistics. It is all about the tails of the distribution. It is used to describe the extreme values in one versus the other tail. It is actually the measure of outliers present in the distribution[[Medium]](https://codeburst.io/2-important-statistics-terms-you-need-to-know-in-data-science-skewness-and-kurtosis-388fef94eeaa)[[Wikipedia](https://en.wikipedia.org/wiki/Kurtosis#Interpretation)]

### PR DESCRIPTION
**Pull request by ritesh99rakesh**
#### References to other Issues or PRs
Fixes #16360 

#### Brief description of what is fixed or changed
Added kurtosis function under stats and tests for it

#### Release Notes

* stats
   * added `kurtosis` method

### CODE REVIEW

**Comment by ritesh99rakesh** [comment]
Please review @Shekharrajak @oscargus @smichr. Regards.

File: sympy/stats/rv_interface.py [hunk 1]
```diff
@@ -208,6 +208,31 @@ def skewness(X, condition=None, **kwargs):
...
+    >>> from sympy import Symbol
+    >>> X = Normal('X', 0, 1)
+    >>> kurtosis(X)
+    3
+    >>> rate = Symbol('lambda', positive=True, real=True, finite=True)
+    >>> Y = Exponential('Y', rate)
+    >>> kurtosis(Y)
+    9
+    """
+    return smoment(X, 4, condition, **kwargs)
```
Comment (oscargus) [inline sympy/stats/rv_interface.py]: Probably you would like to pass `condition` as an optional argument, so `condition=condition`.

File: sympy/stats/rv_interface.py [hunk 2]
```diff
@@ -208,6 +208,31 @@ def skewness(X, condition=None, **kwargs):
     """
     return smoment(X, 3, condition, **kwargs)
 
+def kurtosis(X, condition=None, **kwargs):
+    """
+    Measure of the tailedness(outliers) of the probability distribution
```
Comment (oscargus) [inline sympy/stats/rv_interface.py]: space before ( and full stop (.) at the end.

File: sympy/stats/rv_interface.py (hunk 1 above)
Comment (ritesh99rakesh) [reply sympy/stats/rv_interface.py]: Can I also make the same change for skewness function?

File: sympy/stats/rv_interface.py [hunk 3]
```diff
@@ -208,6 +208,31 @@ def skewness(X, condition=None, **kwargs):
...
+
+    Kurtosis of any univariate normal distribution is 3. Kurtosis less than
+    3 means that the distribution produces fewer and less extreme outliers
+    than the normal distribution
+
+    kurtosis(X) = E( ((X - E(X))/sigma)**4 )
```
Comment (oscargus) [inline sympy/stats/rv_interface.py]: No space after ( and before ). Consider using math (LaTeX) formatting here: 
```
`\operatorname{Kurtosis}(x) = E\left[\left(\frac{X - E(X)}{\sigma}\right)^4\right]`  
```
(with backticks around it)

File: sympy/stats/rv_interface.py [hunk 4]
```diff
@@ -208,6 +208,31 @@ def skewness(X, condition=None, **kwargs):
...
+    >>> from sympy.stats import kurtosis, Exponential, Normal
+    >>> from sympy import Symbol
+    >>> X = Normal('X', 0, 1)
+    >>> kurtosis(X)
+    3
+    >>> rate = Symbol('lambda', positive=True, real=True, finite=True)
+    >>> Y = Exponential('Y', rate)
+    >>> kurtosis(Y)
+    9
+    """
```
Comment (oscargus) [inline sympy/stats/rv_interface.py]: Maybe add

```
References
==========

.. [1] https://en.wikipedia.org/wiki/Kurtosis

```
(properly indented)

**Comment by oscargus** [comment]
Looks good in general. Some formatting comments.

File: sympy/stats/rv_interface.py (hunk 1 above)
Comment (oscargus) [reply sympy/stats/rv_interface.py]: Yes, that would make sense.

**Comment by ritesh99rakesh** [comment]
Please review, @oscargus. And if it looks good, please consider merging it.
I more thing, where should I discuss my idea for the project for GSoC'19. I have finalised my ideas.
Regards

File: sympy/stats/rv_interface.py [hunk 5]
```diff
@@ -216,7 +216,9 @@ def kurtosis(X, condition=None, **kwargs):
     3 means that the distribution produces fewer and less extreme outliers
     than the normal distribution.
 
-    kurtosis(X) = E( ((X - E(X))/sigma)**4 )
+    kurtosis(X) = E(((X - E(X))/sigma)**4)
+
+    .. [1] https://en.wikipedia.org/wiki/Kurtosis
```
Comment (oscargus) [inline sympy/stats/rv_interface.py]: Add this to a section References as in my example. (Just paste it at the end of the documentation string.)

**Comment by oscargus** [comment]
Regarding GSoC I have really no idea, but I guess the mailing list would be the first place and try to get hold of a mentor. I am not involved and have no experience with GSoC.

**Comment by oscargus** [comment]
I think you should add a release note comment, like:
```
* stats
   * added `kurtosis` method
```

**Comment by oscargus** [comment]
Looks good!

**Comment by ritesh99rakesh** [comment]
@asmeurer if this looks good, please merge it.

**Comment by ritesh99rakesh** [comment]
@asmeurer @smichr could you please look into this and guide me what has to be done to merge this PR.

**Pull request closed without merging**